js/main.js

```diff
@@ -0,0 +1,8 @@
+function getRandomArbitrary(min, max, num) {
+  if (min >= max) {
+    alert('Минимальное число не может быть больше максимального!');
+  }
+  return (Math.random() * (max - min) + min).toFixed(num);
```

Comment:
Тут надо привести к числу так например
```suggestion
  return Number((Math.random() * (max - min) + min).toFixed(num));
```